google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -370,8 +505,7 @@ public Row readRow(String tableId, ByteString rowKey) {
    * @throws com.google.api.gax.rpc.ApiException when a serverside error occurs
    */
   public Row readRow(String tableId, String rowKey) {
-    return ApiExceptions.callAndTranslateApiException(
-        readRowAsync(tableId, ByteString.copyFromUtf8(rowKey), null));
+    return readRow(tableId, rowKey, new ReadRowOptions());
```

Comment:
same question here

Reply:
I think this will call `public Row readRow(String tableId, String rowKey, @Nullable ReadRowOptions readRowOptions) {` and the rowKey will be converted later